src/style/AccountPage.less

```diff
@@ -15,12 +15,101 @@
 	}
 }
 
-#account {
-	.navbar {
-		.login-link {
-			color: #AD2016;
-			& a {
-				color: #AD2016;
+.AccountPage {
+
+	.user {
+		img {
+			width: 100%;
+			@media @md-max {
+				width: 25%;
+				margin-bottom: 1rem;
+			}
+		}
+		h1 {
+			font-size: 1.6rem;
+			font-weight: 600;
+		}
+	}
+
+	h1, h2, h3, h4 {
+		font-family: 'Montserrat', sans-serif;
+		font-weight: normal;
+		margin-bottom: 1.3rem; // Overrides Bootstrap default to separate elements.
+		align-self: center;
+	}
+
+	h2,h4 {
+		font-weight: 600;
+		text-transform: uppercase;
+	}
+
+	h2 {
+		color: @gl-light-red;
+	}
+
+	p {
```

Comment:
Section specific overrides for p and a are also to be avoided by default. As above, remove, or promote, or comment. Thanks.